pages/linux/popd.md

```diff
@@ -0,0 +1,15 @@
+# popd
+
+> Remove a directory placed on the directory stack via the pushd shell built-in.
```

Comment:
"via the pushd shell built-in" => "by the `pushd` command" 
To make it consistent with the windows page. And also command sounds better than shell built-in. :laughing: